### PR TITLE
*: cut v0.11.0 release

### PR DESCRIPTION
**Pull request by s-urbaniak**
Signed-off-by: Sergiusz Urbaniak <sergiusz.urbaniak@gmail.com>

### CODE REVIEW

**Comment by s-urbaniak** [comment]
cc @paulfantom @brancz @jan--f

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 0.10.0 / 2021-08-02
```
Comment (paulfantom) [inline CHANGELOG.md:1]: What are the changes for 0.11.0 ?

Comment (s-urbaniak) [reply CHANGELOG.md:1]: oh god

Comment (s-urbaniak) [reply CHANGELOG.md]: fixed

**Review by paulfantom — APPROVED** [review]
lgtm on green

**Comment by paulfantom** [comment]
Seems like we are lacking last release branches. I'll create missing ones, but it would be good to remember about creating them with next releases :)

**Pull request merged**